src/app/component/projects/pages/delete/delete.component.ts

```diff
@@ -33,61 +30,40 @@ import { projectMenuItemActions } from "../details/details.component";
   selector: "app-projects-delete",
   template: `
     <app-form
-      *ngIf="project"
+      *ngIf="models"
       [schema]="{ model: {}, fields: [] }"
-      [title]="'Are you certain you wish to delete ' + project.name + '?'"
+      [title]="'Are you certain you wish to delete ' + getProject().name + '?'"
```

Comment:
why `getProject()` here? It looks an awful lot like a property called with method syntax